extensions/amp-subscriptions/0.1/test/test-entitlement.js

```diff
@@ -44,6 +45,13 @@ describes.realWin('EntitlementClass', {}, () => {
     });
   });
 
+  it('should not return the decrypted key in the json', () => {
+    const raw = 'raw';
+    const entitlement = new Entitlement({source, raw, service, granted,
+      grantReason, dataObject, decryptedDocumentKey: 'decryptedDocumentKey'});
+    expect(entitlement.json().decryptedDocumentKey).to.be.undefined;
```

Comment:
Let's check that the `entitlement.decryptedDocumentKey` field itself is what we provide.

Reply:
done